tensorflow/BUILD

```diff
@@ -15,6 +15,7 @@ exports_files([
 ])
 
 load("//tensorflow:tensorflow.bzl", "tf_cc_shared_object")
+load("//tensorflow:tensorflow.bzl", "transitive_hdrs")
```

Comment:
Where do you use this in this file?

Reply:
oops, good catch. I moved it to another file but forgot to remove the load()